### DISCUSSION TITLE
Linux, 180 fps in opengl for 60 fps in OBS

### DISCUSSION THREAD

**Discussion by ogmkp** [obsproject/obs-studio#11573] [General]
Before starting support tickets or opening an issue ticket, I would like to know if it is normal that the OpenGL managed in OBS displays 3x the fps than the demand required in OBS.
Exemple, for 60 fps in OBS, OpenGL handles 180 fps (linux and nvidia in my case).
How to explain this ratio if it's normal?

I'd like to know if it's normal to have, for example with a simple source in a scene, a render time of 1.7 ms at 60 fps but 0.4 ms at 120 fps?

Thanks.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!

**Reply by ogmkp** [reply]
Don't understand anything about this discussion space, better to say what it's for (nothing).
Issues to report bugs related to the software code I understand, but discussions while there is the forum ...

**Reply by Fenrirthviti** [reply]
Discussions are intended for development questions, not general support inquiries.

**Reply by ogmkp** [reply]
Theres a development section in forum:
https://obsproject.com/forum/#development.19

But ok.

**Reply by Fenrirthviti** [reply]
There's also a development channel in our Discord. Are you implying we can only have one location for things?

The project doesn't want support discussions on GitHub, because it clutters the repo, and we have enough to manage and moderate already when it comes to people asking for support. We have our efforts focused elsewhere for it.

Thanks for your understanding!
